compiler/luci/service/src/Nodes/CircleOutput.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <luci/Service/CircleShapeSignatureInference.h>
+
+namespace luci
+{
+
+ShapeSignature ssinf::Algorithm::visit(const luci::CircleOutput *node)
+{
+  return signature_of_input(node, 0);
```

Comment:
Maybe `signature_of_output` ?

Reply:
Um.... `input` of `signature_of_input` seems confusing for same word `input`
If you intent to use `input` as `argument`, how about rename the function as `signature_of_arg` or `signature_of_argument` ?

Reply:
I will change the name to `signature_of_arg` :)